.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,8 @@
+- id: mythril
+  name: Mythril
+  description: Analyze EVM bytecode with Mythril
+  entry: myth
+  args:
+    - analyze
```

Comment:
isn't this fairly restrictive to just one argument? Mythril has multiple commands

Reply:
Adding `disassemble` might be worth it.

Reply:
yeah... didn't want to add a hook for each possible command ... all subcommands can be selected via setting (=overwriting) `args` on user side... maybe worth to add to the README

Reply:
added to the README